### PR TITLE
Update Enum.xml

### PR DESCRIPTION
**Pull request by leviasan**
## Summary
Fixed invalid information in docs. It's was checked by .NET 6.0.

### CODE REVIEW

File: xml/System/Enum.xml [hunk 1]
```diff
@@ -1961,16 +1961,15 @@ thisInstance And flag = flag
...
         <param name="provider">An object that supplies culture-specific formatting information.</param>
         <summary>Converts the current value to a Boolean value based on the underlying type.</summary>
-        <returns>This member always throws an exception.</returns>
+        <returns>The converted value.</returns>
         <remarks>
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- Currently not implemented. Always throws an exception.
+ Uses <xref:System.Convert> to perform the conversion.
```
Comment (gewarren) [inline xml/System/Enum.xml:1969]:
```suggestion
In .NET 5 and later versions, this method uses <xref:System.Convert> to perform the conversion. In all previous versions of .NET, including .NET Framework, this method always throws an exception.
```

File: xml/System/Enum.xml [hunk 2]
```diff
@@ -1961,16 +1961,15 @@ thisInstance And flag = flag
...
 
  ]]></format>
         </remarks>
-        <exception cref="T:System.InvalidCastException">In all cases.</exception>
```
Comment (gewarren) [inline xml/System/Enum.xml:1973]: This should be retained with a qualifer:

```
        <exception cref="T:System.InvalidCastException">.NET Framework and .NET Core 1.0-3.1 only: In all cases.</exception>
```

File: xml/System/Enum.xml [hunk 3]
```diff
@@ -1961,16 +1961,15 @@ thisInstance And flag = flag
       <Docs>
         <param name="provider">An object that supplies culture-specific formatting information.</param>
         <summary>Converts the current value to a Boolean value based on the underlying type.</summary>
-        <returns>This member always throws an exception.</returns>
+        <returns>The converted value.</returns>
```
Comment (gewarren) [inline xml/System/Enum.xml:1964]:
```suggestion
        <returns>The converted value. (In .NET Framework and .NET Core 1.0-3.1, this method always throws an exception.)</returns>
```

**Review by gewarren — COMMENTED** [review]
It looks like it's been implemented at least since [.NET 5](https://github.com/dotnet/runtime/blob/cf258a14b70ad9069470a108f13765e0e5988f51/src/libraries/System.Private.CoreLib/src/System/Enum.cs#L1113-L1116). If that's the case, then I made some suggestions on the wording since the docs are reused across all versions.

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=8301) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=8301) before we can accept your contribution.<br/><br/>:x: Leviasan [sign now](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=8301)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/dotnet-api-docs?pullRequest=8301) it.</sub>

**Comment by gewarren** [comment]
@Leviasan The wording looks good - I'd just like a System.Runtime subject matter expert to confirm that the documented behavior is correct.

**Pull request closed without merging**